test/fake_modules/package_import_map/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "imports":{
+    "#simple": "simple-resolved",
+    "#subpath": "subpath-resolved/subpath",
+    "#conditional": {
+      "node": "conditional-node",
+      "types": "@types/conditional"
```

Comment:
I don't see `"types"` in the list of conditions that are handled https://nodejs.org/api/packages.html#conditional-exports

Reply:
That's from TypeScript, e.g. https://www.typescriptlang.org/docs/handbook/esm-node.html

Reply:
Oooh, gotcha, thanks